### DISCUSSION TITLE
Open bookmark's link in new tab

### DISCUSSION THREAD

**Discussion by IM71** [Floorp-Projects/Floorp#1141] [Feature Request]
Please consider adding a option that can set open bookmark's link in new tab as default with left click.

Thank you

**Comment by rguhr** [comment]
In `about:config` there are these two options to manipulate the behavior for bookmark links especially:

```
browser.tabs.loadBookmarksInBackground
browser.tabs.loadBookmarksInTabs
```

**Reply by IM71** [reply]
Got it. Thanks.
